application-templates/starter-typescript/tsconfig.json

```diff
@@ -1,3 +1,9 @@
 {
-  "extends": "@commercetools-frontend/application-config/tsconfig-mc-app.json"
+  "extends": "@commercetools-frontend/application-config/tsconfig-mc-app.json",
+  "compilerOptions": {
+    "typeRoots": ["@types", "@types-extensions", "node_modules/@types"],
+    // Allow tsc to resolve dependencies of `@types` packages in the way pnpm sets up the `node_modules`.
+    "preserveSymlinks": false
```

Comment:
Sorry to be late to the party but just wanted to understand if this change would have any impact in the package manager a developer using the starter template can use.

Reply:
There should be no issues as far as I know, even when using Yarn.

However, we can keep an eye on it in case someone reports some issues that might relate to it.

Thanks for bringing this up!